web/public/js/controllers/dojo-events-list-controller.js

```diff
@@ -70,6 +70,19 @@
           event.for = $translate.instant(userType);
           events.push(event);
         });
+        events.sort(function (eventA, eventB) {
```

Comment:
tempted to say we should move it to event utils, as it's probably going to be used for my-tickets ?